### PR TITLE
Remove Atom (retired) from the proposed editors

### PR DESCRIPTION
**Pull request by nap84**
While looking over the course material, I clicked the link to [Atom](https://atom.io/) since I was curious about it as an alternative to VS Code. I dicovered this editor has been sunset to be able to focus efforts on VS Code. I therefore suggest removing it from the suggested editors and adjusting the respective sentence accordingly.

### CODE REVIEW

**Review by sharilaster — APPROVED** [review]
Hi @nap84, that's a great catch, and this update looks good to me.  I'll wait a bit to see if any other lesson maintainers want to review before merging.

**Comment by morskyjezek** [comment]
I agree. This should be updated and the proposed change looks good to me.

**Comment by morskyjezek** [comment]
This seems like a practical change and I don't see any objections, so I'll go ahead an merge. Thanks you @nap84 !

**Pull request merged**
